homeassistant/components/resolution_center/websocket_api.py

```diff
@@ -37,6 +42,46 @@ def ws_dismiss_issue(
     connection.send_result(msg["id"])
 
 
+@websocket_api.websocket_command(
+    {
+        vol.Required("type"): "resolution_center/fix_issue",
+        vol.Required("domain"): str,
+        vol.Required("issue_id"): str,
+    }
+)
+@websocket_api.async_response
+async def ws_fix_issue(
+    hass: HomeAssistant, connection: websocket_api.ActiveConnection, msg: dict
+) -> None:
+    """Start a flow to an issue."""
+    flow_manager: ResolutionCenterFlowManager = hass.data[DOMAIN]["flow_manager"]
+    result = await flow_manager.async_init(
+        msg["domain"], data={"issue_id": msg["issue_id"]}
+    )
+    connection.send_message(
+        websocket_api.result_message(msg["id"], _prepare_result_json(result))
+    )
+
+
+@websocket_api.websocket_command(
+    {
+        vol.Required("type"): "resolution_center/fix_issue_confirm",
+        vol.Required("flow_id"): str,
+        vol.Optional("user_input"): dict,
+    }
+)
+@websocket_api.async_response
+async def ws_fix_issue_confirm(
```

Comment:
This is wrong. You need to expose the full data entry flow views to interact with the flow. The flows don't have to be just a confirm.

Reply:
https://github.com/home-assistant/core/blob/dev/homeassistant/components/config/config_entries.py

```python
from homeassistant.helpers.data_entry_flow import (
    FlowManagerIndexView,
    FlowManagerResourceView,
)
```

Reply:
The name is misleading, `ws_fix_issue_confirm` allows passing input to a fix flow which is in progress, not only confirming a flow.

I modeled this based on the mfa flows which are exposed via ws instead of via views.